core/source/org/libreoffice/ide/eclipse/core/wizards/NewUnoProjectWizard.java

```diff
@@ -237,6 +237,9 @@ public IWizardPage getNextPage(IWizardPage pPage) {
                     } else {
                         // Could be null
                         next = mServiceSet.getPage(ServiceWizardSet.SERVICE_PAGE_ID);
+                        if (mMainPage.getChosenLanguage().toString().contains("python")) {
```

Comment:
This is done so that the ServicePage and the Interface page do not show up in the wizard. Currently I feel there is no need to include those pages in the New Project Wizard when the selected language is Python.